lucene/test-framework/src/java/org/apache/lucene/analysis/ValidatingTokenFilter.java

```diff
@@ -50,6 +56,9 @@
   private final OffsetAttribute offsetAtt = getAttribute(OffsetAttribute.class);
   private final CharTermAttribute termAtt = getAttribute(CharTermAttribute.class);
 
+  // record all the Tokens seen so they can be dumped on failure
```

Comment:
Update comment to `record last MAX_DEBUG_TOKENS...`?

Reply:
oh good, thanks. I pushed an edit